src/Team/team.manager.ts

```diff
@@ -1,17 +1,34 @@
-import { getCurrentPlayerNumberInRepo, getPlayersNumberInTeamRepo, updateNumber } from './team.repository';
+import { getCurrentPlayerNumberInRepo, getIdofAllPlayersrepo, getPlayersId, getPlayersNumberInTeamRepo, updateNumber } from './team.repository';
 
 export async function getPlayersNumberInTeamManger(teamInt: number) {
   const checkNumber = Array.from(await getPlayersNumberInTeamRepo(teamInt));
   //   const playersNumber: number[] = checkNumber[0].playersNumber;
   return checkNumber;
 }
+
+export async function getPlayersIdInTeamManger(teamInt: number) {
+  const checkNumber = Array.from(await getPlayersNumberInTeamRepo(teamInt));
+  //   const playersNumber: number[] = checkNumber[0].playersNumber;
+  return checkNumber;
+}
+
+export async function getIds(teamInt: number) {
```

Comment:
It is hard to understand what this function does. why do you use an array of arrays([][])? What does it return? The names are not informative. 
Add documentration and change the names of the variables.